mne/epochs.py

```diff
@@ -583,7 +583,7 @@ def _detrend_offset_decim(self, epoch, verbose=None):
         return epoch
 
     def iter_evoked(self):
-        """Iterate over Evoked objects with nave=1
+        """Iterate epochs as a sequence of Evoked objects with n_epochs=1
```

Comment:
True.  But `nave` is, IMO, simply not clear:

- It is not a parameter of the `Evoked` constructor.
- It does not follow a convention seen elsewhere in the code and docs of using `n_` as a prefix for something that is a count of items (n_samples, n_epochs, n_components, n_tapers, n_rows, n_cols, n_vertices, n_times...).
- In its `__repr__` method [it prints `nave` as `n_epochs`](https://github.com/mne-tools/mne-python/blob/master/mne/evoked.py#L126).  And docstrings, like `__repr__` output, are for humans to read.
- It is [an extant (though somewhat rare) word of English](http://www.merriam-webster.com/dictionary/nave), and might easily be mistaken for obscure neuroimaging term.